.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        go: [ '1.16', '1.17' ]
+        go: [ '1.16', '1.17', '1.18' ]
```

Comment:
We can remove the tests for Go 1.16. We only support the most recent 2 releases of Go.

Reply:
We're keeping 1.16 minimum until COPR can use 1.17 on all their supported platforms

Reply:
These are just CI tests though, not releases, right? I don't think we need to triple our CI tests for COPRs. (I really do not like the idea of external distribution platforms holding us back. The added CI tests do slow things down.)

Reply:
It doesn't slow things down, the tests are run in parallel.